hazelcast-jet-distribution/pom.xml

```diff
@@ -106,121 +88,34 @@
             <artifactId>hazelcast-jet</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
-        <!-- workaround, otherwise javadoc from core module is not included-->
         <dependency>
-            <groupId>com.hazelcast.jet</groupId>
-            <artifactId>hazelcast-jet-core</artifactId>
-            <version>${project.parent.version}</version>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>${log4j.version}</version>
         </dependency>
         <dependency>
             <groupId>com.hazelcast.jet</groupId>
             <artifactId>hazelcast-jet-avro</artifactId>
             <version>${project.parent.version}</version>
+            <classifier>jar-with-dependencies</classifier>
```

Comment:
all of these jar-with-dependencies also shade Jet, so we have 5 copies of Jet inside lib folder